src/main/app/src/app/shared/pipes/datum.pipe.ts

```diff
@@ -13,6 +13,10 @@ export class DatumPipe implements PipeTransform {
     }
 
     transform(value: Date | moment.Moment | string, dateFormat?: string): any {
+        if (value && value.toString().includes('[UTC]')) {
```

Comment:
Even constante maken van [UTC] in regel hierboven

Reply:
Aangepast